### DISCUSSION TITLE
How to open an application window on a specific monitor.

### DISCUSSION THREAD

**Discussion by ershuaili** [tauri-apps/tauri#9243] [Q&A]
My computer has multiple monitors. Every time I run an application, I want it to be displayed in a window that does not have the application. Or can I specify the window to be displayed on the monitor with a specified name through the monitor name?

**Comment by dolev146** [comment]
upvote for that lets make a function for that
@FabianLars  can you teach me how and we will make it a standard ? is that thing a possibility or you don't want this responsibility on tauuri ?
